widgets/text/options.go

```diff
@@ -174,3 +176,16 @@ func ScrollKeys(up, down, pageUp, pageDown keyboard.Key) Option {
 		opts.keyPgDown = pageDown
 	})
 }
+
+// The default for the maximum buffer length within a content area
+// -1 sets as no limit
+const (
+	DefaultMaxContent = -1
+)
+
+// MaxContent - Limits the maximum content within text widget buffer
```

Comment:
This is the only public place where this feature is documented. Would you mind expanding the comment and explaining the behavior `Text` has when this is set?

It would also be good to specify whether we are limiting the number of runes, or the number of cells. I assume this is limiting the number of runes to be displayed and therefore the number of cells may be variable (since some unicode runes take multiple cells).

Reply:
I've increased the commenting but it may require further clarification here. If my read of the code is correct, `t.content` is the cells so the limits should be based on the cells?

Reply:
That is correct, `t.content` is indeed in cells. How would you feel about renaming the option and its default to make it obvious? Something like:

`MaxContentCells` and `DefaultMaxContentCells`.

We could also mention that the number represents the maximum amount of terminal cells the content can take and note that some runes might take multiple cells for clarity.